stdlib/public/core/Codable.swift

```diff
@@ -67,6 +67,19 @@ public protocol CodingKey {
     init?(intValue: Int)
 }
 
+extension CodingKey {
+    /// A textual representation of this key.
+    public var description: String {
+        let intValue = self.intValue?.description ?? "nil"
+        return "\(String(describing: type(of: self)))(stringValue: \"\(stringValue)\", intValue: \(intValue))"
```

Comment:
You can just use `\(type(of: self))` instead of `\(String(describing: type(of: self)))`